GalaxyZooTouchTable/GalaxyZooTouchTable/Models/GalaxyExampleFactory.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Threading.Tasks;
+using System.Windows.Media.Imaging;
+
+namespace GalaxyZooTouchTable.Models
+{
+    enum GalaxyType
+    {
+        Smooth,
+        Features,
+        Star
+    }
+
+    public class GalaxyExample
+    {
+        BitmapImage ImageOne { get; }
+        BitmapImage ImageTwo { get; }
+        BitmapImage ImageThree { get; }
+        string Title { get; }
+        string Description { get; }
+        string Description2 { get; }
+    }
+
+    static class GalaxyExampleFactory
+    {
+        public static GalaxyExample Create(GalaxyType type)
+        {
+            switch (type)
+            {
+                case GalaxyType.Smooth:
+                    return new SmoothGalaxy();
+                case GalaxyType.Features:
+                    return new FeaturesGalaxy();
+                case GalaxyType.Star:
+                    return new StarGalaxy();
+                default:
+                    return new StarGalaxy();
+            }
+        }
+
+        private class SmoothGalaxy : GalaxyExample
```

Comment:
Not sure if this is worth changing. But I think it would make more sense if you gave `GalaxyExample` a constructor that looks like.

```
private GalaxyExample(GalaxyType type) {
	switch(type) {
	// Do all the specific value assignments
	}
}
``` 

This way you can get rid of all the subclasses that don't actually extend the class